autopush/main_argparse.py

```diff
@@ -141,6 +141,11 @@ def _obsolete_args(parser):
     parser.add_argument('--storage_read_throughput', help="OBSOLETE")
     parser.add_argument('--storage_write_throughput', help="OBSOLETE")
 
+    parser.add_argument('--fcm_enabled', help="OBSOLETE")
```

Comment:
(re above) or was this the intention?

```suggestion
    parser.add_argument('--fcm_auth', help="OBSOLETE")
```